plugins/nf-validation/src/testResources/nextflow_schema.json

```diff
@@ -17,6 +17,7 @@
                     "format": "file-path",
                     "mimetype": "text/csv",
                     "pattern": "^\\S+\\.(csv|tsv|yaml)$",
+                    "schema": "src/testResources/schema_input.json",
```

Comment:
We have `nextflow_schema_with_samplesheet.json` which will validate samplesheets, `nextflow_schema.json)` should only be for testing pipeline parameters, so it's better to not add the `schema` keyword here.

Reply:
Aaah okay I was confused why it wasn't in there because there were tests to validate the samplesheets using that schema

Reply:
Should be fixed now!